OSDOCS-6187: Update the OCP 4.14 Welcome page (#66276)
openshift/openshift-docs: welcome/index.adoc

== Cluster installer activities
Explore the following {product-title} installation tasks:

- **xref:../installing/index.adoc#ocp-installation-overview[{product-title} installation overview]**: Depending on the platform, you can install {product-title} on installer-provisioned or user-provisioned infrastructure. The {product-title} installation program provides the flexibility to deploy {product-title} on a range of different platforms.

- **xref:../installing/installing_alibaba/preparing-to-install-on-alibaba.adoc#preparing-to-install-on-alibaba[Install a cluster on Alibaba]**: On Alibaba Cloud, you can install {product-title} on installer-provisioned infrastructure. This is currently a Technology Preview feature only.

> - **xref:../installing/installing_aws/preparing-to-install-on-aws.adoc#preparing-to-install-on-aws[Install a cluster on AWS]**: On AWS, you can install {product-title} on either installer-provisioned infrastructure or user-provisioned infrastructure.

Reviewer: ```suggestion
- **xref:../installing/installing_aws/preparing-to-install-on-aws.adoc#preparing-to-install-on-aws[Install a cluster on AWS]**: On AWS, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

> - **xref:../installing/installing_aws/installing_azure/preparing-to-install-on-azure#preparing-to-install-on-azure[Install a cluster on Azure]**: On Microsoft Azure, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.

Reviewer 2: ```suggestion
- **xref:../installing/installing_azure/preparing-to-install-on-azure.adoc#preparing-to-install-on-azure[Install a cluster on Azure]**: On Microsoft Azure, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

> - **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.

Reviewer 2: ```suggestion
- **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack Hub, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

> - **xref:../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.

Reviewer 2: The other links go to the "Preparing to install" page, this one goes to "Configuring a GCP project". Think you want:
```suggestion
- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

> - **xref:../installing/installing_azure_stack_hub//installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.

Reviewer 2: ```suggestion
- **xref:../openshift-enterprise/latest/installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[AWS] or
xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]
on Microsoft Azure, you can install a cluster. Also consider xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]

- **Install a private cluster**: If your cluster does not require external
internet access, you can install a private cluster on
xref:../installing/installing_aws/installing-aws-private.adoc#installing-aws-private[AWS],
xref:../installing/installing_azure/installing-azure-private.adoc#installing-aws-private[Azure],
xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[GCP], or

> xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[{ibmcloudVPCRegProductName} Cloud VPC]. Internet access is still required to access the cloud APIs and installation media.

Reviewer 2: ```suggestion
xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[{ibmcloudVPCRegProductName}]. Internet access is still required to access the cloud APIs and installation media.
```

xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[AWS] or
xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]

> on Microsoft Azure, you can install a cluster. Also consider in xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]

Reviewer 2: ```suggestion
on Microsoft Azure, you can install a cluster. Also consider xref:../installing/installing_gcp/installing-gcp-shared-vpc.adoc#installation-gcp-shared-vpc-prerequisites_installing-gcp-shared-vpc[Installing a cluster on GCP into a shared VPC]
```

endif::[]

ifndef::openshift-rosa,openshift-dedicated,openshift-dpu,microshift[]
== Developer activities
Develop and deploy containerized applications with {product-title}. {product-title} is a platform for developing and deploying containerized applications. Read the following {product-title} documentation, so that you can better understand {product-title} functions:

- **xref:../architecture/understanding-development.adoc#understanding-development[Understand {product-title} development]**: Learn the different types of containerized applications, from simple containers to advanced Kubernetes deployments and Operators.

- **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.

> - **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating applications using the *Developer perspective]**: Use the *Developer* perspective in the {product-title} web console to easily create and deploy applications.

Reviewer 2: ```suggestion
- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Creating applications using the Developer perspective]**: Use the *Developer* perspective in the {product-title} web console to easily create and deploy applications.
```

- **xref:../applications/odc-viewing-application-composition-using-topology-view#odc-viewing-application-composition-using-topology-view.adoc#odc-creating-applications-using-developer-perspective[Viewing application composition using the *Topology* view]**: Use the *Topology* view to visually interact with your applications, monitor status, connect and group components, and modify your code base.

- **Use the xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer* perspective]** in the {product-title} web console to
xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[create and deploy applications].

- **Use the xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[*Topology*] view**

> to see your applications, monitor status, connect and group components, and modify your code base.

Reviewer 2: These seem to be redundant with the two right above

ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
== Cluster administrator activities

Manage machines, provide services to users, and follow monitoring and logging reports. Read the following {product-title} documentation, so that you can better understand {product-title} functions:

- **xref:../architecture/architecture.adoc#architecture-overview-architecture[Understand {product-title} management]**: Learn about components
of the {product-title} {product-version} control plane. See how {product-title} control plane and worker nodes are managed and updated through the xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machine-api-overview_creating-machineset-aws[Machine API] and xref:../architecture/control-plane.adoc#operators-overview_control-plane[Operators].

> - **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].

Reviewer 2: ```suggestion
- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrator, you can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].
```

> - **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].

Reviewer 2: Same link twice, with almost identical text. Probably just need one.
```suggestion
- **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can enable cluster capabilities that were disabled prior to installation.
```

> - **xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enable cluster capabilities]**: As a cluster administrators, you can can enable cluster capabilities that were disabled prior to installation. For more information, see xref:../post_installation_configuration/enabling-cluster-capabilities.adoc#enabling-cluster-capabilities[Enabling cluster capabilities].

Author: Good catch. No time for the can-can!
